src/Presentation/Startup.cs

```diff
@@ -24,6 +27,8 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<Schema>();
             services.AddScoped<Detection.Creating.IResolver, Detection.Creating.Resolver>();
             services.AddScoped<Detection.Fetching.IResolver, Detection.Fetching.Resolver>();
+            services.AddTransient<IGetDetection, GetDetection>();
+            services.AddTransient<IDetectionRepository, DetectionRepository>();
```

Comment:
Should we consider using the `Scoped` registrations instead?

Reply:
Done